ONSdigital/dp-permissions-api
# Update endpoint name to /roles/{id} and remove hello endpoint

### What

- Update get /roles/{id} endpoint to include the s and align it with the Swagger spec
- Remove the hello world endpoint

### How to review

- Check the endpoint name has been changed to /roles/{id} 
- Check hello world endpoint has been removed

### Who can review

Anyone
